platforms/jvm/language-java/build.gradle.kts

```diff
@@ -30,7 +31,6 @@ dependencies {
     api(project(":files"))
     api(project(":hashing"))
     api(project(":language-jvm"))
-    api(project(":messaging"))
```

Comment:
💭 Here is one of the projects that can now depend only on `:serialization` instead of on `:messaging` (and the native dependencies it brings in).